bin/interpreter.sh

```diff
@@ -206,15 +210,12 @@ if [[ ! -z "$ZEPPELIN_IMPERSONATE_USER" ]]; then
 fi
 
 if [[ -n "${SPARK_SUBMIT}" ]]; then
-    if [[ -n "$ZEPPELIN_IMPERSONATE_USER" ]] && [[ "$ZEPPELIN_IMPERSONATE_SPARK_PROXY_USER" != "false" ]];  then
```

Comment:
By removing this `ZEPPELIN_IMPERSONATE_SPARK_PROXY_USER` you disable the feature that was introduced with #1840 (`# export ZEPPELIN_IMPERSONATE_SPARK_PROXY_USER  #Optional, by default is true; can be set to false if you don't want to use --proxy-user option with Spark interpreter when impersonation enabled`), is that intended? If so, we will need to update the doc.